app/views/categories/_category.html.erb

```diff
@@ -0,0 +1,31 @@
+<div id="<%= dom_id category %>">
+  <p class="my-5">
+    <strong class="block font-medium mb-1">Name:</strong>
+    <%= category.name %>
+  </p>
+
+  <div>
+    <strong class="block font-medium mb-1">Products:</strong>
+    <% products = Product.where(category_id: category.id) %>
+    <% if products.any? %>
+      <ul>
+        <% products.each do |product| %>
```

Comment:
i believe thst we dont need create another one page for each category, we will create filter with categories on /product page